src/Appointment/Admin/AppointmentsAdmin.js

```diff
@@ -66,29 +66,29 @@ class AppointmentsAdmin extends React.Component {
       searchDate: e.target.value,
     });
   }
-  
+
   handleFilterChange = (event) => {
     this.setState({ filter: event.target.value });
   };
 
   updateAppointment() {
     if (this.state.searchDate) {
       const newAppointments = this.state.appointments.filter((req) => {
-        return moment(req.schedule.date.date).isSame(this.state.searchDate); 
+        return moment(req.schedule.date.date).isSame(this.state.searchDate);
       });
       this.setState({ filterAppointments: newAppointments });
     } else if (this.state.dayValue) {
```

Comment:
If we change it to
`else if (this.state.dayValue != 9999) {`
it'll solve the issue I face (see my other comment)

Please test it and see if this is a solution, thanks!